src/index.html

```diff
@@ -9,8 +9,29 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./page.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page">
+    <form
+      data-qa="big"
+      class="page__form"
+    >
+      <input
+        data-qa="keypress"
+        class="page__input page__input--big"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+      <div class="page__search-logo page__search-logo--big"></div>
```

Comment:
Don't use div to add images. You can set the background image directly to the input or use pseudoelement for that